test/fmf/tests/utils.sh

```diff
@@ -31,25 +102,56 @@ uninstall_server() {
 }
 
 start_service_manufacturer() {
+  if [ "$USE_TLS" = true ]; then
+    configure_tls manufacturer
+  fi
```

Comment:
This will be done/configured in `generate_certs` based on the service protocol, so not needed for manufacturer or any other service